### ISSUE TITLE
Bridges between 2 instances not working

### ISSUE THREAD

**Issue by juliolrmonteiro** [holochain/holochain-rust#1209]
Hey there,

When I create 2 instances on a holochain conductor, I can't seem to be able to bridge between them using a `hdk::call`.

Here's my call code:
```rust
    let holofuel_address = match hdk::call(
        "fuel-instance",
        "transactions",
        "".into(),
        "request",
        json!({
            "from": "xxx-undefined_address-xxx",
            "amount": price_per_unit.unwrap(), // TODO: use the real value
            "notes": "service log", // TODO: put some nice notes
            "deadline": "" // TODO: use some actual dealine
        }).into()
    ) {
        Ok(json) => serde_json::from_str(&json.to_string()).unwrap(),
        Err(e) => return Err(e)
    };
```

And here's my conductor config:
https://github.com/Holo-Host/servicelogger/blob/master/conductor-config.toml

**Comment by zippy** [comment]
Looks like you are using the same storage for both instances in that config.  This needs to be different, there's a bug in that the current consistency checker doesn't warn you about that.

**Comment by Connoropolous** [comment]
@juliolrmonteiro this get resolved? if so, can you close?

**Comment by juliolrmonteiro** [comment]
Great, that is fixed, thanks @zippy and @lucksus.
I just had to replace `"fuel-instance"` to `"holofuel-bridge"`. Will try to get this setup documented to help others.
